renamePackage.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python
+
+# Usage renamePackage.sh package
+# Ex: renamePackage.sh com.demo.mobile
+
+import os, sys
+import shutil
+import platform
+from functools import reduce
+
+stuffToRemove = [".gradle", ".git", ".idea", "build", "app/build", ".iml", "local.properties"]
+dirChar = '\\' if platform.system() == 'Windows' else '/'
```

Comment:
Could you use `os.sep` (https://docs.python.org/2/library/os.html#os.sep) instead?